Scripts Enhacement

cristiancmoises opened the pull request:
Suggestions for Enhancing Code Readability and Maintainability

cristiancmoises committed:
Update copy-language-pack.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update download-language-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update prepare-flatpak-release.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update remove-failed-jobs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update_ff.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update_newtab.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update check-rc-response.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update copy-language-pack.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update download-language-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update remove-failed-jobs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update-en-US-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update-en-US-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update_ff.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update_newtab.py

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

mr-cheffy commented:
There seem to be some git conflicts. After that, it's ready for merge!

cristiancmoises committed:
Update update-en-US-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update download-language-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update-en-US-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update download-language-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update download-language-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update update-en-US-packs.sh

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises commented:
> There seem to be some git conflicts. After that, it's ready for merge!
Ok.

mr-cheffy committed:
Merge branch 'dev' into patch-2

Signed-off-by: mr. m  <91018726+mr-cheff@users.noreply.github.com>

mr-cheffy commented:
Checks are failing, you might need to lint them

cristiancmoises committed:
Merge branch 'zen-browser:dev' into patch-2

mr-cheffy commented:
https://github.com/zen-browser/desktop/actions/runs/12612016593/job/35148299725?pr=4162

cristiancmoises committed:
Merge branch 'zen-browser:dev' into patch-2

cristiancmoises committed:
Merge branch 'zen-browser:dev' into patch-2

cristiancmoises committed:
Update code-linter.yml

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises committed:
Update sign.ps1

Refactored Repeated Logic: Created functions for downloading artifacts, signing files, moving files, and creating tar archives.

Error Handling: Added checks for file existence before moving files and implemented verbose logging.

Parameter Validation: Used ValidateNotNullOrEmpty for parameters.

Improved Path Handling: Used Join-Path for constructing file paths

Signed-off-by: Cristian Cezar Moisés <ethicalhacker@riseup.net>

cristiancmoises commented:
@mauro-balades Done. Thank you.

mr-cheffy committed:
Discard changes to .github/workflows/code-linter.yml

mr-cheffy commented:
Please do fix the formatting/linter issues.

mr-cheffy committed:
Discard changes to scripts/copy-language-pack.sh

mr-cheffy committed:
Discard changes to scripts/download-language-packs.sh
